dvc/project.py

```diff
@@ -195,7 +199,38 @@ def _check_cyclic_graph(graph):
         if cycles:
             raise CyclicGraphError(cycles[0])
 
-    def add(self, fname, recursive=False):
+    @staticmethod
+    def _check_can_commit(stage, entry, force):
```

Comment:
should it be part of `stage` class?

Reply:
Yes, will fix. Thank you! :slightly_smiling_face: